starter-code/MongooseMovies/models/celebrity.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// create celebrity schema
+const CelebritySchema = new Schema ({
+  name: String,
+  occupation: String,
+  catchphrase: String
+});
+
+// create celebrity model
+const Celebrity = mongoose.model('Celebrity', CelebritySchema); // don't fully understand this
```

Comment:
Here we are using the mongoose model constructor, to create our Celebrity model. The mongoose.model method receives two arguments: the name of the model, and the schema, which is the "skeleton" of the model.
The result of this constructor is an object of class Model which we are saving in a variable and exporting.
Later, we are using methods of this model in order to create new celebs in the DB, do queries on the collection etc. 

Consider "Celebrity.findById", "Celebrity.findByIdAndRemove" etc. These are all methods of the model object which we are creating here, and they do the queries for us. Let me know if you want to discuss this in person.